code/modules/mob/living/carbon/xenomorph/castes/queen/castedatum_queen.dm

```diff
@@ -171,3 +172,63 @@
 
 	// *** Queen Abilities *** //
 	queen_leader_limit = 4
+
+//same stats as ancient
+/datum/xeno_caste/queen/primordial
+	caste_desc = "A fearsome Xeno hulk of titanic proportions. Nothing can stand in it's way."
+	primordial_message = "Destiny bows to our will as the universe trembles before us."
+	upgrade = XENO_UPGRADE_FOUR
+	// *** Melee Attacks *** //
+	melee_damage = 23
+
+	// *** Speed *** //
+	speed = -0.3
+
+	// *** Plasma *** //
+	plasma_max = 1200
+	plasma_gain = 70
+
+	// *** Health *** //
+	max_health = 500
+
+	// *** Evolution *** //
+	upgrade_threshold = 100
```

Comment:
this doesn't work like that, the threshold needs to be greater or equal to the other, because it carries over upgrade and evolution